.gitignore

```diff
@@ -1,3 +1,4 @@
 .tox/
 __pycache__/
 *.pyc
+.idea
```

Comment:
It is not preferable to add patterns related to IDE, please remove .idea

Reply:
removed as suggested